# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR ORGANIZATION
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: \n"
"POT-Creation-Date: 2016-07-19 08:10+0200\n"
"PO-Revision-Date: 2016-07-19 08:10+0200\n"
"Last-Translator: \n"
"Language-Team: \n"
"Language: es\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: pygettext.py 1.5\n"
"X-Generator: Poedit 1.8.8\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:63
msgid "Update failed!"
msgstr "¡La actualización ha fallado!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:63
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4174
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4183
msgid "Could not connect to %s"
msgstr "No se puede conectar a %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:261
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:263
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:265
msgid "Operating System not supported: %s"
msgstr "Sistema Operativo no soportado: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:280
msgid "Copy old appdata to new folder failed!"
msgstr ""
"¡Ha fallado la copia de datos de aplicación antigua a la carpeta nueva!"

# No translation needed
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:280
msgid "Error: def win_pre1_check_app_dir()"
msgstr "Error: def win_pre1_check_app_dir()"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:291
msgid "Could not create app_dir: %s"
msgstr "No se puede crear app_dir: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:321
msgid "Select User-ID failed!"
msgstr "¡Ha fallado la selección de la ID de usuario!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:387
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:390
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:393
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:396
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:399
msgid "Error: '%s' not found!"
msgstr "Error: '%s' ¡No encontrado!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:416
msgid ""
"Could not remove lock file.\n"
"File itself locked because another oVPN Client instance running?"
msgstr ""
"No se pudo eliminar el archivo de bloqueo.\n"
"¿Está el archivo bloqueado por otra instancia de cliente de oVPN "
"ejecutándose?"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:455
msgid ""
"Creating API-DIRS\n"
"%s \n"
"%s \n"
"%s \n"
"%s \n"
"%s failed!"
msgstr "¡La creación de API-DIRS %s %s %s %s %s ha fallado!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:457
msgid "Creating config Folders failed"
msgstr "Ha fallado la creación de las carpetas de configuración"

# No translation needed
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:682
msgid "Error: def read_options_file"
msgstr "Error: def read_options_file"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:682
msgid "Read config file failed!"
msgstr "¡Ha fallado la lectura del archivo de configuración!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:815
msgid ""
"Error reading your Interfaces from netsh.exe. Contact support@ovpn.to with "
"Error-ID:\n"
"ADAPTERS[1]=(%s)\n"
msgstr ""
"Error al leer sus Interfaces desde netsh.exe. Contacte con support@ovpn.to "
"con la Error-ID:\n"
"ADAPTERS[1]=(%s)\n"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:850
msgid ""
"Could not read your Network Interfaces! Please install OpenVPN or check if "
"your TAP-Adapter is really enabled and driver installed."
msgstr ""
"¡No se pudo leer las Interfaces de Red! Por favor, instale OpenVPN o "
"compruebe que su Adaptador TAP está habilitado e instalado el controlador."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:881
msgid "No OpenVPN TAP-Windows Adapter found!"
msgstr "¡No se ha encontrado el Adaptador OpenVPN TAP-Windows!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:895
msgid ""
"Multiple TAPs found!\n"
"\n"
"Please select your TAP Adapter!"
msgstr "¡Múltiples TAPs encontrados! ¡Por favor, seleccione su Adaptador TAP!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:911
msgid ""
"No OpenVPN TAP-Adapter found!\n"
"Please install openVPN!\n"
"URL1: %s\n"
"URL2: %s"
msgstr ""
"¡No se ha encontrado ningún Adaptador TAP!\n"
" ¡Por favor, instale openVPN!\n"
" URL1: %s\n"
" URL2: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:916
msgid "Invalid characters in '%s'"
msgstr "Carácteres inválidos en '%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:932
msgid "Choose your External Network Adapter!"
msgstr "¡Seleccione su Adaptador de Red Externo!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:950
msgid "No Network Adapter found!"
msgstr "¡No se encontró ningún adaptador de red!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:965
msgid "Please select your User-ID!"
msgstr "¡Por favor, seleccione su ID de Usuario!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1012
msgid "Disconnect %s"
msgstr "Desconectar %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1016
msgid "Connect to %s"
msgstr "Conectar a %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1024
msgid "Remove AutoConnect: %s"
msgstr "Eliminar AutoConnect: %s"

# To Set accepts multiple variants like:
# poner
# ajustar
# fijar
# colocar
# etc...
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1028
msgid "Set AutoConnect: %s"
msgstr "Establecer AutoConnect %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1043
msgid "[enabled]"
msgstr "[habilitado]"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1045
msgid "[disabled]"
msgstr "[deshabilitado]"

# If Server is plural, it would be 'de los Servidores'
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1046
msgid "Load Server Information %s"
msgstr "Cargar información del Servidor %s"

# Same as above: ' de los Servidores'
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1059
msgid "Set Server-View Size [%sx%s]"
msgstr "Ajuste tamaño de vista del Servidor [%sx%s]"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1066
msgid "Update every: %s seconds"
msgstr "Actualizar cada: %s segundos"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1084
msgid "Change DNS"
msgstr "Cambiar DNS"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1158
msgid "Set Primary DNS"
msgstr "Establezca DNS Primario"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1168
msgid "Set Secondary DNS"
msgstr "Establezca DNS Secundario"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1329
msgid "Checking for Updates!"
msgstr "¡Comprobando actualizaciones!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1332
msgid "Downloading Configurations..."
msgstr "Descargando configuraciones..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1335
msgid "Requesting Certificates..."
msgstr "Solicitando certificados..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1338
msgid "Please wait... Certificates requested from backend! (%s)"
msgstr ""
"Solicitado los certificados desde el back-end (%s), por favor, espere..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1346
msgid "Downloading Certificates..."
msgstr "Descargando certificados..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1349
msgid "Extracting Configs and Certs..."
msgstr "Extrayendo configuraciones y certificados..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1354
msgid "Disconnected! Have a nice and anonymous day!"
msgstr "¡Desconectado! ¡Que tenga un bonito y anónimo día!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1374
msgid "Connecting to %s"
msgstr "Conectando a %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1383
msgid "Testing connection to %s"
msgstr "Comprobando conexión a %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1388
msgid "Connection to %s unstable or failed!"
msgstr "Conexión inestable o fallada a %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1403
msgid "Connected to %s [%s]:%s (%s) [ %s ] (%s / %s ms)"
msgstr "Conectado a %s [%s]:%s (%s) [ %s ] (%s / %s ms)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1565
msgid " [ disconnect ]"
msgstr "[ desconectar ]"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1608
msgid "Close Servers"
msgstr "Cerrar Servidores"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1611
msgid "Servers"
msgstr "Servidores"

# Also:
# Salir de la cuenta
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1621
msgid "Close Account"
msgstr "Cerrar Cuenta"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1624
msgid "Account"
msgstr "Cuenta"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1634
msgid "Close Settings"
msgstr "Cerrar Ajustes"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1636
msgid "Settings"
msgstr "Ajustes"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1649
msgid "About"
msgstr "Acerca de"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1657
msgid "Quit"
msgstr "Salir"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1718
msgid "Error: Update Timeout"
msgstr "Error: Tiempo expirado para actualización"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1718
msgid ""
"Update took too long...aborted!\n"
"Please retry in few minutes..."
msgstr ""
"¡La actualización ha llevado demasiado tiempo...abortado!\n"
"Por favor, reinténtelo en unos minutos..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1736
msgid "Certificates and Configs updated!"
msgstr "¡Los certificados y configuraciones han sido actualizadas!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1736
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1749
msgid "oVPN Update OK!"
msgstr "¡Actualización de oVPN correcta!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1739
msgid "Extraction failed!"
msgstr "¡La extracción ha fallado!"

# No translation needed
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1739
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1741
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1744
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1746
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1754
msgid "Error: def inThread_timer_check_certdl"
msgstr "Error: def inThread_timer_check_certdl"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1741
msgid "Failed to download certificates"
msgstr "Falló la descarga de certificados"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1744
msgid "Failed to request certificates!"
msgstr "¡Falló la solicitud de certificados!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1746
msgid "Failed to download configurations!"
msgstr "¡Falló la descarga de configuración!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1749
msgid "No update needed!"
msgstr "¡No es necesaria ninguna actualización!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1754
msgid "Failed to check for updates!"
msgstr "¡Falló la comprobación de actualizaciones!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1999
msgid "oVPN Server - %s"
msgstr "Servidor oVPN - %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2071
msgid "oVPN Server [ %s ]"
msgstr "Servidor oVPN [ %s ]"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2207
msgid "oVPN Account - %s"
msgstr "Cuenta oVPN  - %s"

# Alternative: 'Cargar información de cuenta'
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2227
msgid "Settings -> Updates: 'Load Account Info' [disabled]"
msgstr "Ajustes -> Actualizaciones: 'Cargar ajustes de cuenta' [deshabilitado]"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2229
msgid "No data loaded! Retry in few seconds..."
msgstr "¡No se ha cargado ningún dato! Reinténtelo en unos segundos..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2326
msgid "oVPN Settings - %s"
msgstr "Ajustes oVPN - %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2359
msgid " Security "
msgstr " Seguridad"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2376
msgid " Options "
msgstr "  Opciones"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2390
msgid " Updates "
msgstr " Actualizaciones"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2400
msgid "Restore Firewall Backups\n"
msgstr "Restaurar copias de seguridad del cortafuegos\n"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2402
msgid " Backups "
msgstr "Copias de seguridad"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2410
msgid "Use Windows Firewall (default: ON) "
msgstr "Usar Cortafuegos de Windows (por defecto: encendido)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2440
msgid "TAP-Adapter block outbound (default: OFF)"
msgstr "Bloque Saliente de Adaptador de TAP (por defecto: apagado)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2471
msgid "Block Internet on Disconnect or Quit (default: ON)"
msgstr "Bloquear Internet al desconectar o cerrar (por defecto: encendido)"

# Instead 'FW' it can be 'Cortafuegos'
# already very long, client supposed to know what is FW.
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2499
msgid "Disable FW question on Quit (default: OFF)"
msgstr "Deshabilitar pregunta acerca de FW al salir (por defecto: encendido)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2527
msgid "Clear Rules on Connect (default: OFF)"
msgstr "Reglas claras al conectar (por defecto: apagado)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2547
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2577
msgid "Could not export Windows Firewall Backup!"
msgstr "¡No se pudo exportar copia de seguridad de cortafuegos de Windows!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2547
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2577
msgid "Error: Windows Firewall Backup failed"
msgstr "Error: Falló la copia de seguridad del cortafuegos de Windows"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2557
msgid "Backup on Start / Restore on Quit (default: OFF)"
msgstr ""
"Copia de seguridad al Inicio / Restaurar al Salir (por defecto: apagado)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2587
msgid "DNS Leak Protection (default: ON)"
msgstr "Protección de fuga de DNS (por defecto: encendido)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2617
msgid "Disable '%s' on Disconnect (default: OFF)"
msgstr "Deshabilitar '%s' al Desconectar (por defecto: apagado)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2655
msgid "Update Configs on Start (default: OFF)"
msgstr "Actualizar configuraciones al Iniciar (por defecto: apagado)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2680
msgid "Load Account Info (default: OFF)"
msgstr "Cargar información de la cuenta (por defecto: apagado)"

# Not clear if Server is single or plural
# Used single here.
# In plural would be 'de los servidores' instead 'del servidor' only.
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2714
msgid "Load Server Info (default: OFF)"
msgstr "Cargar información del Servidor (por defecto: apagado)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2749
msgid "Debug Mode (default: OFF)"
msgstr "Modo de depuración (por defecto: apagado)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2765
msgid "Debug Mode Enabled"
msgstr "Modo de depuración habilitado"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2765
msgid ""
"Logfile:\n"
"'%s'"
msgstr ""
"Archivo de registro:\n"
" '%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2774
msgid "Select Network Adapter"
msgstr "Seleccione Adaptador de Red"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2785
msgid "Use IPv4 Entry Server with Exit to IPv4 (standard)"
msgstr "Selección: Servidor entrante con salida a IPv4 (estándar)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2788
msgid "Use IPv4 Entry Server with Exits to IPv4 + IPv6"
msgstr "Selección: Servidor entrante con salida a IPv4 + IPv6"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2813
msgid "Change App Theme (default: ms-windows)"
msgstr "Cambiar el tema de la aplicación (por defecto: ms-windows)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2846
msgid "Change App Font Size (default: 9)"
msgstr "Cambiar tamaño de fuente de la aplicación (por defecto: 9)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2892
msgid "Change Language"
msgstr "Cambiar idioma"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2928
msgid "Disable About and Quit on Connection (default: ON)"
msgstr "Deshabilitar Acerca de y Salir al conectar (por defecto: encendido)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2949
msgid "Normal Config Update"
msgstr "Actualización de la configuración normal"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2958
msgid "Forced Config Update"
msgstr "Actualización forzada de la configuración"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2967
msgid "Reset API-Login"
msgstr "Reiniciar API-Login"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3200
msgid "Please wait!"
msgstr "¡Por favor, espere!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3200
msgid "Update is running."
msgstr "Actualización en curso."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3275
msgid "Error: Certs not found!"
msgstr "Error: ¡No se encontraron los certificados!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3275
msgid "Files missing: '%s'"
msgstr "Archivos faltantes: '%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3303
msgid "Could not start Windows Firewall!"
msgstr "¡No se pudo iniciar el cortafuegos de Windows!"

# No translation needed
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3303
msgid "Error: def inThread_spawn_openvpn_process"
msgstr "Error: def inThread_spawn_openvpn_process"

# Not sure what it means
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3618
msgid "Error: DNS restore 2nd"
msgstr "Error: Restaurar DNS secundario"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3618
msgid "Error: Restore Secondary DNS to %s failed."
msgstr "Error: Restaurar DNS secundario si falló %s."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3621
msgid "Error: DNS restore 1st"
msgstr "Error: Restaurar DNS primario"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3621
msgid "Error: Restore Primary DNS to %s failed."
msgstr "Error: Restaurar DNS primario si falló %s."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3684
msgid "def win_netsh_read_dns_to_backup: failed!"
msgstr "def win_netsh_read_dns_to_backup: ¡fallado!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3717
msgid ""
"Error:\n"
"SSL Root Certificate for %s not loaded %s"
msgstr ""
"Error:\n"
"Certificado Raíz SSL para %s no cargado %s"

# No translation needed
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3717
msgid "Error: SSL CA Cert #1"
msgstr "Error: SSL CA Cert #1"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3720
msgid ""
"Error:\n"
"Invalid SSL Root Certificate for %s in:\n"
"'%s'\n"
"hash is:\n"
"'%s'\n"
"but should be '%s'"
msgstr ""
"Error:\n"
"Certificado Raíz SSL inválido para %s en:\n"
"'%s'\n"
"hash es:\n"
"'%s'\n"
"pero debería ser '%s'"

# No translation needed
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3720
msgid "Error: SSL CA Cert #2"
msgstr "Error: SSL CA Cert #2"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3723
msgid ""
"Error:\n"
"SSL Root Certificate for %s not found in:\n"
"'%s'!"
msgstr "Error: ¡No se encontró Certificado Raíz SSL para %s en: '%s'!"

# No translation needed
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3723
msgid "Error: SSL CA Cert #3"
msgstr "Error: SSL CA Cert #3"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3912
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3936
msgid "Error: netsh.exe not found!"
msgstr "Error: ¡No se encontró netsh.exe!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3951
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3975
msgid "Error: route.exe not found!"
msgstr "Error: ¡No se encontró route.exe!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3991
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4003
msgid "ipconfig.exe not found!"
msgstr "¡No se encontró ipconfig.exe!"

# In case if it is an installer.
# otherwise it is 'Ajustes oVPN.to'
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4038
msgid "oVPN.to Setup"
msgstr "Asistente de configuración oVPN.to"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4039
msgid "Enter your oVPN.to Details"
msgstr "Introduzca los detalles de su oVPN.to"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4047
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4051
msgid "User-ID:"
msgstr "ID de usuario:"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4061
msgid "API-Key:"
msgstr "Clave API:"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4174
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4183
msgid "Error: Update failed"
msgstr "Error: ¡La actualización ha fallado!"

# 'Servidores' in case of plural
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4224
msgid "Server Window Size"
msgstr "Tamaño ventana Servidor"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4228
msgid "Width (pixel):"
msgstr "Ancho (pixeles):"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4232
msgid "Height (pixel):"
msgstr "Alto (pixeles):"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4236
msgid ""
"Enter width and height\n"
"\n"
"Leave blank for default"
msgstr ""
"Introduzca ancho y alto\n"
"\n"
"Dejar vacío para valores por defecto"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4282
msgid "Load Data every X seconds"
msgstr "Cargar Datos cada X segundos"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4286
msgid "seconds:"
msgstr "segundos:"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4333
msgid "Switched to IPv4+6"
msgstr "Conmutado a IPv4+6"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4333
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4350
msgid ""
"Changed Option:\n"
"\n"
"Use 'Forced Config Update' to get new configs!\n"
"\n"
"You have to join 'IPv6 Beta' on https://%s to use any IPv6 options!"
msgstr ""
"Opción cambiada:\n"
"\n"
"¡Use 'Actualización forzada de la configuración' para obtener nuevas "
"configuraciones!\n"
"\n"
"¡Tiene que ingresar en 'IPv6Beta' en https://%s para usar cualquier opción "
"IPv6!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4350
msgid "Switched to IPv6+4"
msgstr "Conmutado a IPv6+4"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4398
msgid "No API-Key!"
msgstr "¡No hay Clave API!"

# No translation needed
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4398
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4428
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4439
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4451
msgid "Error: def API_REQUEST"
msgstr "Error: def API_REQUEST"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4428
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4451
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4718
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4775
msgid "Invalid User-ID or API-Key or Account expired!"
msgstr "¡ID de usuario inválida o Clave API o Cuenta expirada!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4439
msgid "API requests on: %s failed!"
msgstr "¡Las solicitudes de la API fallaron en: %s!"

# No translation needed
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4538
msgid "Error: def move_appdata"
msgstr "Error: def move_appdata"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4538
msgid "Move Userdata to new folder: '%s'"
msgstr "Mover Datos de usuario a carpeta nueva: '%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4574
msgid "Could not read Servers Remote-IP:Port from config: %s"
msgstr ""
"No se pudo leer la IP Remota de los Servidores:Puerto de la configuración: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4583
msgid "Could not read Servers Protocol from config: %s"
msgstr "No se pudo leer Protocolo de los Servidores desde la configuración: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4595
msgid "Could not read Servers Cipher from config: %s"
msgstr ""
"No se pudo leer el Cifrado de los Servidores desde la configuración: %s"

# No translation needed
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4621
msgid "Error: def load_ovpn_server"
msgstr "Error: def load_ovpn_server"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4621
msgid "Load openVPN Server failed"
msgstr "Falló la carga del Servidor openVPN"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4797
msgid "Platform '%s' not supported"
msgstr "Plataforma '%s' no soportada"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4797
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4848
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4852
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4936
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4989
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4992
msgid "Error!"
msgstr "¡Error!"

# If setup refers to some windows installer.
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4813
msgid ""
"openVPN Setup downloaded and hash verified OK!\n"
"\n"
"Please start setup from file:\n"
"'%s'\n"
"\n"
"Verify GPG with:\n"
"'%s'"
msgstr ""
"¡Descargado ajustes de openVPN con verificación hash exitosa!\n"
"\n"
"Por favor, inicie el asistente de configuración desde el archivo:\n"
"'%s'\n"
"\n"
"Verifique GPG con:\n"
"'%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4815
msgid ""
"openVPN Setup downloaded but hash verify failed!\n"
"Please install openVPN!\n"
"URL1: %s\n"
"URL2: %s"
msgstr ""
"¡Descargado ajustes de openVPN pero falló la verificación hash!\n"
"¡Por favor, instale openVPN!\n"
"URL1: %s\n"
"URL2: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4822
msgid ""
"Install OpenVPN %s (%s) (%s)\n"
"\n"
"Starting download (~1.8 MB) from:\n"
"'%s'\n"
"to\n"
"'%s'\n"
"\n"
"Please wait..."
msgstr ""
"Instale OpenVPN %s (%s) (%s)\n"
"\n"
"Comenzando descarga (~1.8MB) desde:\n"
"'%s'\n"
"a\n"
"'%s'\n"
"\n"
"Por favor, espere..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4822
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4888
msgid "Setup: openVPN"
msgstr "Asistente de configuración: openVPN"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4848
msgid ""
"Invalid File hash: %s !\n"
"localhash = '%s'\n"
"but should be = '%s'"
msgstr ""
"¡Hash de archivo inválido: %s !\n"
"localhash = '%s'\n"
"pero debería ser = '%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4852
msgid "Failed remove file: %s"
msgstr "No se pudo eliminar el archivo: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4888
msgid ""
"OpenVPN not found!\n"
"\n"
"Please select openvpn.exe on next window!\n"
"\n"
"If you did not install openVPN yet: click cancel on next window!"
msgstr ""
"¡No se encontró OpenVPN!\n"
"\n"
"¡Por favor, seleccione openvpn.exe en la siguiente ventana!\n"
"\n"
"¡Si aún no ha instalado OpenVPN haga click en cancelar en la siguiente "
"ventana!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4889
msgid "Select openvpn.exe or Cancel to install openVPN"
msgstr "Seleccione openvpn.exe o Cancelar para instalar openVPN"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4936
msgid ""
"WARNING! Failed to verify files in\n"
"'%s'\n"
"\n"
"Uninstall openVPN and restart oVPN Client Software!\n"
"\n"
"Or install openVPN from URL:\n"
"%s[debug self.LAST_FAILED_CHECKFILE = '%s']"
msgstr ""
"¡Advertencia! Falló la verificación de archivos en\n"
"'%s'\n"
"¡Desinstale openVPN y reinicie el cliente oVPn!\n"
"\n"
"O instale openVPN desde URL:\n"
"%s[debug self.LAST_FAILED_CHECKFILE = '%s']"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4941
msgid "Could not detect openVPN Version!"
msgstr "¡No se pudo detectar la versión de openVPN!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4965
msgid "Could not find openVPN"
msgstr "¡No se encuentra openVPN!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4989
msgid "Invalid Hash: '%s'! is '%s' != '%s'"
msgstr "¡Hash no válido: '%s'! es '%s' !='%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4992
msgid "Invalid content '%s' in '%s'"
msgstr "Contenido no válido '%s' en '%s'"

# If arch means architecture!
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:5009
msgid "arch '%s' not supported!"
msgstr "¡arquitectura '%s' no soportada!"

# also: 'Cerrar cliente oVPN.to'
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:5217
msgid "Quit oVPN.to Client"
msgstr "Salir del cliente oVPN.to"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:5221
msgid "Do you really want to quit?"
msgstr "¿Está seguro de salir?"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:5287
msgid "Firewall Settings"
msgstr "Configuración del cortafuegos"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:5291
msgid ""
"Restore previous firewall settings?\n"
"\n"
"Press 'YES' to restore your previous firewall settings!\n"
"Press 'NO' to set profiles to 'blockinbound,blockoutbound'!"
msgstr ""
"¿Restaurar ajustes previos del cortafuegos?\n"
"\n"
"¡Presione 'SI' para restaurar sus ajustes previos del cortafuegos!\n"
"¡Presione 'NO' para fijar los perfiles a 'blockinbound,blockoutbound'!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:5294
msgid ""
"Allow outgoing connection to internet?\n"
"\n"
"Press 'YES' to set profiles to 'blockinbound,allowoutbound'!\n"
"Press 'NO' to set profiles to 'blockinbound,blockoutbound'!"
msgstr ""
"¿Permitir conexión saliente hacia Internet?\n"
"\n"
"¡Presione 'SI' para fijar los perfiles a 'blockinbound,allowoutbound'!\n"
"¡Presione 'NO' para fijar los perfiles a 'blockinbound,allowoutbound'!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:5353
msgid "Error"
msgstr "Error"

#~ msgid "Security Settings\n"
#~ msgstr "Ajustes de seguridad\n"

#~ msgid "Options\n"
#~ msgstr "Opciones\n"

#~ msgid "Updates\n"
#~ msgstr "Actualizaciones\n"

#~ msgid "Change App Theme"
#~ msgstr "Cambiar tema de la App"

# "Preferencias" is also widely used and valid :)
#~ msgid "Options"
#~ msgstr "Opciones"

#~ msgid "IPv6 Options"
#~ msgstr "Opciones IPv6"

#~ msgid "Select: IPv4 Entry Server with Exit to IPv4 (standard)"
#~ msgstr "Selección: Servidor entrante con salida a IPv4 (estándar)"

#~ msgid "Select: IPv4 Entry Server with Exits to IPv4 + IPv6"
#~ msgstr "Selección: Servidor entrante con salida a IPv4 + IPv6"

#~ msgid "Firewall"
#~ msgstr "Cortafuegos"

#~ msgid "Restore Firewall Backups"
#~ msgstr "Restaurar copias de seguridad de Cortafuegos"

#~ msgid "Disable FW question on Quit (default: ON)"
#~ msgstr ""
#~ "Deshabilitar pregunta acerca de FW al salir (por defecto: encendido)"
